tests/tensorflow/nightly/classifier-efficientnet.libsonnet

```diff
@@ -144,6 +144,11 @@ local gpus = import "templates/gpus.libsonnet";
   local v3_32 = tpu_common {
     accelerator: tpus.v3_32,
   },
+  local reserved = {
```

Comment:
This might be better as a mixin in the `tpus.libsonnet` file. The usage would be `... + tpus.reserved. We used to have something similar for preemptible

Reply:
good idea - will fix